packages/css/types/tests.ts

```diff
@@ -28,11 +28,3 @@ css`
   position: relative;
   top: ${'20px'};
 `
-
-// $ExpectError
-css(() => 'height: 300px;')
```

Comment:
These no longer fail, not sure that is an issue? It's a FunctionInterpolation returning a SerialisedInterpolation isn't it?

Reply:
I think this should fail (not sure 100% yet), because of this:
https://github.com/emotion-js/emotion/blob/a835b89ce8448c590d931ea03e21e506e245c927/packages/serialize/src/index.js#L197
You are right about what this currently is, but seems like css and styled should have slightly different Interpolation type. The pattern here should be allowed for styled, but not so much for css

Reply:
Switching the type from Interpolation to CSSInterpolation in the css module fixed the tests.. I didn't change any of this so can only assume it was failing before?

Reply:
Might be as TS tests were not run on CI. Would be great to fix it here though.

Reply:
Cool!